### PR TITLE
feat(exchange): replace OTRS by Support module

### PR DESCRIPTION
**Pull request by FredericEspiau**
Signed-off-by: Frédéric Espiau <frederic.espiau@corp.ovh.com>

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -11859,7 +11859,7 @@ ovh-ngstrap@^4.0.2:
   resolved "https://registry.yarnpkg.com/ovh-ngstrap/-/ovh-ngstrap-4.0.2.tgz#059291bfb9f59b78df991e476dd2ede0507cb3c9"
   integrity sha1-BZKRv7n1m3jfmR5HbdLt4FB8s8k=
 
-ovh-ui-angular@^3.9.6:
+ovh-ui-angular@^3.9.5, ovh-ui-angular@^3.9.6:
```
Comment (frenautvh) [inline yarn.lock]: you didn't add any dependencies, you shouldn't have changes in the yarn.lock

Comment (FredericEspiau) [reply yarn.lock]: I know, what do you want me to do, I guess someone didn't `yarn install` ¯\_(ツ)_/¯

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6361.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

File: packages/manager/modules/exchange/package.json [hunk 2]
```diff
@@ -18,7 +18,6 @@
   "peerDependencies": {
     "@ovh-ux/ng-ovh-export-csv": "^1.0.0",
     "@ovh-ux/ng-ovh-http": "^4.0.1-beta.0",
-    "@ovh-ux/ng-ovh-otrs": "^9.0.1",
```
Comment (marie-j) [inline packages/manager/modules/exchange/package.json]: This means we'll need to deprecate this module ?

Comment (FredericEspiau) [reply packages/manager/modules/exchange/package.json]: Yes :)

**Pull request merged**